### PR TITLE
unipicker: init at 767571c

### PR DESCRIPTION
**Pull request by karthikiyengar**
###### Motivation for this change
Introduces a new package for unipicker: https://github.com/jeremija/unipicker

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -4404,6 +4404,12 @@
       fingerprint = "8992 44FC D291 5CA2 0A97  802C 156C 88A5 B0A0 4B2A";
     }];
   };
+  kiyengar = {
```
Comment (SuperSandro2000) [inline maintainers/maintainer-list.nix:4407]: Please put this into a separate commit with the message ``maintainers: add kiyengar``.

File: pkgs/applications/misc/unipicker/default.nix [hunk 2]
```diff
@@ -0,0 +1,36 @@
+{ stdenv, fetchFromGitHub, lib, fzf, xclip }:
+
+stdenv.mkDerivation rec {
+   pname = "unipicker";
+   version = "767571c87cdb1e654408d19fc4db98e5e6725c04";
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/unipicker/default.nix:5]:
```suggestion
   version = "unstable-2018-07-10";
```

File: pkgs/applications/misc/unipicker/default.nix [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+   src = fetchFromGitHub {
+      owner = "jeremija";
+      repo = pname;
+      rev = version;
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/unipicker/default.nix:10]:
```suggestion
      rev = "767571c87cdb1e654408d19fc4db98e5e6725c04";
```

File: pkgs/applications/misc/unipicker/default.nix [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+      substituteInPlace unipickerrc --replace "/usr/local" "$out"
+   '';
+
+   makeFlags = [
+      "PREFIX=$(out)"
+      "DESTDIR=$(out)"
+   ];
+
+   meta = with lib; {
+    description = "A CLI utility for searching unicode characeters by description and optionally copying them to clipboard.";
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/unipicker/default.nix:30]:
```suggestion
    description = "A CLI utility for searching unicode characeters by description and optionally copying them to clipboard";
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]
Please squash all the package related commits.

**Comment by karthikiyengar** [comment]
@SuperSandro2000 - Thanks for the review :smile:! I've addressed your comments.

**Comment by SuperSandro2000** [comment]
@karthikiyengar please take a look at the failing test https://logs.nix.ci/?key=nixos/nixpkgs.117511&attempt_id=63a17b30-0957-4365-b326-252a808635c2 and maybe disable if it does not work in the nix sandbox.

**Comment by lukegb** [comment]
@ofborg build unipicker unipicker.passthru.tests
